fdbserver/include/fdbserver/WorkerInterface.actor.h

```diff
@@ -725,10 +725,12 @@ struct InitializeCommitProxyRequest {
 	Version recoveryTransactionVersion;
 	bool firstProxy;
 	ReplyPromise<CommitProxyInterface> reply;
+	EncryptionAtRestMode encryptMode;
```

Comment:
Should this be `Optional` given encryptMode database config might not be set at all?

Reply:
By the time we make this request the value is always available (the request is made after we set the promise) here: https://github.com/apple/foundationdb/blob/main/fdbserver/ClusterRecovery.actor.cpp#L1028 (promise gets set here: https://github.com/apple/foundationdb/blob/main/fdbserver/ClusterRecovery.actor.cpp#L968). This is not used in the codepath for provisional proxies

Reply:
Makes sense. Thanks